DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/_FaqCard.cshtml

```diff
@@ -0,0 +1,61 @@
+<div class="searchable-element nhsuk-expander-group word-break" id="@Model.FaqId-card">
+
+  <details class="nhsuk-details nhsuk-expander">
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-grid-row">
+        <div class="nhsuk-grid-column-one-quarter">
+          <div class="nhsuk-tag tag card-tag--grey">
+            @Model.TargetGroup
+          </div>
+        </div>
+        <div class="nhsuk-grid-column-one-quarter">
+          <div class="nhsuk-tag tag card-tag--grey">
+            @Model.Published
+          </div>
+        </div>
+      </div>
+    </div>
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text" id="@Model.FaqId-name">
+        <span class="searchable-element-title" name="name">@Model.QText</span>
+      </span>
+    </summary>
+
+    <div class="nhsuk-details__text">
+      <div class="nhsuk-u-margin-bottom-4">
+        <span class="searchable-element-content">@Html.Raw(Model.AHtml)</span>
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-one-quarter">
+            <p class="nhsuk-u-font-weight-bold">Created</p>
+          </div>
+          <div class="nhsuk-grid-column-one-quarter">
+            <p>@Model.CreatedDate</p>
+          </div>
+        </div>
+        <div class="nhsuk-grid-row">
+          <div class="nhsuk-grid-column-one-quarter">
+            <p class="nhsuk-u-font-weight-bold">Weighting</p>
+          </div>
+          <div class="nhsuk-grid-column-one-quarter">
+            <p>@Model.Weighting</p>
+          </div>
+        </div>
+      </div>
+
+      <div>
+        <a class="nhsuk-button nhsuk-button"
+           role="button">
+          Edit
+        </a>
```

Comment:
This button seems to have "nhsuk-button" class twice. 

Additionally, you might want to fiddle with the margins on these. When using smaller screen sizes, the default nhsuk-buttons add a huge margin underneath, which can be overwritten with the `nhsuk-u-margin-bottom-2` or something similar, depending on whatever size looks good.

Reply:
![image](https://user-images.githubusercontent.com/88332001/151829358-4ecfa9ca-ded9-44f8-bfce-02c3a464fa65.png)

Reply:
cheers this does look better